src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.servise.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String FIRST_LINE = "fruit, quantity";
+
+    @Override
+    public String report() {

```

Comment:
We should have verb in the method name
```suggestion
    public String generateReport() {

```